_pages/clients/dispute.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Dispute
+permalink: /dispute/
+web: https://ethicnology.github.io/dispute/#/
+github: ethicnology/dispute
+instances:
+- name: Dispute on GitHub Pages 
+  url: https://ethicnology.github.io/dispute/#/
+  comment: 
+tags:
+- social
+- client
+- debate
```

Comment:
The product doesn't look like anything at this point. Not worth introducing a new tag.

```suggestion
```